Better formatting in netcdf definition tables

gavinmacaulay opened the pull request:
Modified formatting of netcdf definition tables that should make them easier to read. Have done the short tables only. Need to check that this doesn't break how @cyrilleponcelet creates and edits the tables. If ok, will do the long tables too.

Also adds in the equation numbers that were missing compared to the ICES CRR pdf version.

gavinmacaulay committed:
Add equqtion numbers and eqution cross-referencing. Now matches the ICES CRR 341 pdf version.

gavinmacaulay committed:
Format the tables a little more nicely.

cyrilleponcelet commented:
Hi @gavinmacaulay , given the modification I will have no problem to adapt by table generator. That's OK with me, thanks for taking me into account.
Note that it is nicer in the generated document but not in github viewer (we are using it more than the official pdf), have you consider to add on the top of each asciidoctor file the definition of var and attr formatting ? :
____
:var: {nbsp}{nbsp}{nbsp}{nbsp}
:attr: {var}{var}
____

Have a nice day
Cyrille

gavinmacaulay commented:
Good point about the github viewer. I'll sort that out too.

gavinmacaulay committed:
Update the remaining tables to make them easier to read
Make the substitutions for indenting in the table an include in each table adoc so that viewing individual files works better (such as via github's asciidoctor viewer).

gavinmacaulay committed:
Github doesn't use includes in .adoc files, so for the tables put the substitutions into each table file.

gavinmacaulay committed:
Fix a table caption that had become merged with a paragraph

gavinmacaulay committed:
Make some table cells extend into adjacent cells to reduce wrapping inside cells.

gavinmacaulay committed:
Where possible, make parameter cells take up the entire row
